infra/modules/service/variables.tf

```diff
@@ -67,3 +73,65 @@ variable "db_vars" {
   })
   default = null
 }
+
+variable "container_env_vars" {
+  type        = list(map(string))
+  description = "Additional environment variables to pass to the container definition"
+  default     = []
+}
+
+variable "container_secrets" {
+  type        = list(map(string))
+  description = "AWS secrets to pass to the container definition"
+  default     = []
+}
+
+variable "aws_services_security_group_id" {
+  type        = string
+  description = "Security group ID for VPC endpoints that access AWS Services"
+}
+
+
+variable "container_read_only" {
+  type        = bool
+  description = "Whether the container root filesystem should be read-only"
+  default     = true
+}
+
+#-------------------
+# Healthcheck
+#-------------------
+
+variable "healthcheck_path" {
+  type        = string
+  description = "The path to the application healthcheck"
+  default     = "/health"
+}
+
+variable "healthcheck_type" {
+  type        = string
+  description = "Whether to configure a curl or wget healthcheck. curl is more common. use wget for alpine-based images"
+  default     = "wget"
+  validation {
+    condition     = contains(["curl", "wget"], var.healthcheck_type)
+    error_message = "choose either: curl or wget"
+  }
+}
+
+variable "healthcheck_matcher" {
+  type        = string
+  description = "The response codes that indicate healthy to the ALB"
+  default     = "200-299"
+}
```

Comment:
I remember going through a fair bit of this healthcheck customization for flowable (https://github.com/navapbc/pfml-starter-kit-app/commit/ea2f9064a6884f94434e03a4ff2ee63f675511a5), though I can't remember why we had that set to `200-401` there. I think probably because we couldn't disable the security on the endpoint, but at least it's not a 500 (or 404) error so *something* is working.